content/docs/command-reference/exp/remove.md

```diff
@@ -27,8 +27,12 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `--queue` - remove all experiments that haven't been run yet (defined via
   `dvc exp run --queue`).
 
-- `-A`, `--all` - remove all experiments that have been run. Use `--queue` to
-  remove queued ones.
+  > ⚠️ `dvc exp remove --queue` is now an alias for `dvc queue remove --queued`.
+  > The `--queue` option will likely be deprecated and removed in a future DVC
+  > release. Refer to the `dvc queue` documentation for more details.
```

Comment:
What about failed experiments? Does `dvc exp remove --queue` remove them?

Reply:
We didn't save failed experiments at all before, previously failed runs were re-added to the queue. So they were removed by `dvc exp remove --queue`.

But with the new behavior I don't think they should be removed with `dvc exp remove --queue` (since failed runs are handled separately and are not re-queued)